lti_consumer/lti_consumer.py

```diff
@@ -270,6 +280,48 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         default="",
         scope=Scope.settings
     )
+
+    # LTI 1.3 fields
+    lti_version = String(
+        display_name=_("LTI Version"),
+        scope=Scope.settings,
+        values=[
+            {"display_name": "LTI 1.1/1.2", "value": "lti_1p1"},
+            {"display_name": "LTI 1.3", "value": "lti_1p3"},
+        ],
+        default="lti_1p1",
```

Comment:
So just to be totally pedantic here - when this xblock's code is updated underneath an existing LTI 1.1 XBlock, this lti_version field will not have a value set, and will therefore default to `lti_1p1` for any/all existing LTI XBlocks in the system?  I.e., just sanity checking that there's a clean, seamless upgrade path for existing LTI 1.1 content.

Reply:
@davidjoy Yes, that's exactly what happens.
I've tested it locally like this:
1. Installed master version of LTI XBlock
2. Created a new block using this test tool:  `https://lti.tools/saltire/tp` .
3. Checked LMS and Studio that the launch worked.
4. Installed the new block version, checked that it was set to use the 1.1 version on the previously created block.
5. Checked that the launch worked on both LMS & Studio.